jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy

```diff
@@ -165,11 +163,15 @@ def test_suites = { suites,option ->
                         bin/pd-server -name=pd3 --data-dir=pd3 --client-urls=http://127.0.0.1:2399 --peer-urls=http://127.0.0.1:2398 -force-new-cluster &> pd3.log &
                         bin/tikv-server --pd=127.0.0.1:2399 -s tikv3 --addr=0.0.0.0:20190 --advertise-addr=127.0.0.1:20190 --advertise-status-addr=127.0.0.1:20185 -C tikv.toml -f  tikv3.log &
             
-                        cd session
-                        export log_level=error
-                        go install gotest.tools/gotestsum@latest
-                        gotestsum --format standard-verbose --junitfile "junit-report.xml" -- -with-tikv -pd-addrs=127.0.0.1:2379,127.0.0.1:2389,127.0.0.1:2399 -timeout 20m -vet=off ${option} '${suites}'
-                        #go test -with-tikv -pd-addrs=127.0.0.1:2379 -timeout 20m -vet=off
+                        if [ -d "${suites}" ]
+                        then
+                            cd ${suites}
+                            export log_level=error
+                            go install gotest.tools/gotestsum@latest
+                            gotestsum --format standard-verbose --junitfile "junit-report.xml" -- -with-tikv -pd-addrs=127.0.0.1:2379,127.0.0.1:2389,127.0.0.1:2399 -timeout 20m -vet=off
```

Comment:
`-with-tikv` is correct? (for pessimistictest and realtikvtest)

test ci link: https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_check_2/detail/tidb_ghpr_check_2/77471/pipeline/78

Reply:
You're right that tag name changed. Push a new commit to fix.